### ISSUE TITLE
Making simple changes to documentation fails

### ISSUE THREAD

**Issue by paulehoffman** [Legrandin/pycryptodome#848]
I made a couple of small (but important) changes to a doc file, but the build failed. See https://github.com/Legrandin/pycryptodome/actions/runs/12307672552

**Comment by Legrandin** [comment]
Yes, that was related to github actions not supporting macos13 anymore (unrelated to what the change could have been).
I have updated the build configuration and actions should work now.
